spec/Section 2 -- Language.md

```diff
@@ -166,9 +166,9 @@ characters to support interoperation with as many other systems as possible.
 
 ## Query Document
 
-Document : Definition+
+Document :: Definition+
```

Comment:
The `::` is semantically meaningful - it distinguishes between character level rules and token level rules.